--- kea-dhcp-ddns.conf	2024-05-13 04:15:43
+++ kea-dhcp-ddns-new.conf	2024-05-13 04:15:38
@@ -23,7 +23,7 @@
   "port": 53001,
   "control-socket": {
       "socket-type": "unix",
-      "socket-name": "/tmp/kea-ddns-ctrl-socket"
+      "socket-name": "/run/kea/kea-dhcp-ddns-ctrl.sock"
   },
   "tsig-keys": [],
   "forward-ddns" : {},
@@ -44,7 +44,7 @@
                 // - syslog (logs to syslog)
                 // - syslog:name (logs to syslog using specified name)
                 // Any other value is considered a name of the file
-                "output": "/data/log/kea-ddns.log"
+                "output": "stdout"
 
                 // Shorter log pattern suitable for use with systemd,
                 // avoids redundant information
